app/konnect/dev-portal/customization/appearance.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Appearance
+no_version: true
+---
+
+The Dev Portal can be customized by those with Admin roles via the Appearance settings in the {{site.konnect_short_name}} Admin UI. To access the Appearance settings, click **Dev Portal** then **Appearance**.
+
+Here, you have the ability to modify the following:
+
+* Logos: default logo and fav icon
+* Home Page Header: welcome message, primary header, and header image
+* Themes
```

Comment:
How about we take out "Themes" from this list? They are actually pre-configured look and feels that can then be modified with all the options we bring up in the list. I am not sure what the best way is to explain "Themes" at the moment since they do not provide much value at the moment. Maybe we should just ignore any explanation for now and bring content in as soon as we have the new "Default" theme look and feel. WDYT?

Reply:
Sure, I would say it's not uncommon to use the term theme in the way Dev Portal uses that word. On the Insomnia-side we have some "themes" as well with minor modifications available (font, highlight color) globally, so I think "theme" works here. "Theme" generally means "change the bulk of appearance". I'll take it out.

Reply:
"Themes" has been removed from the list.